otel-extensions/src/main/java/org/hypertrace/agent/otel/extensions/HypertraceGlobalIgnoreMatcher.java

```diff
@@ -52,7 +52,9 @@ public Result classloader(ClassLoader classLoader) {
     }
 
     String name = classLoader.getClass().getName();
-    if (name.startsWith("com.singularity.") || name.startsWith("com.yourkit.")) {
+    if (name.startsWith("com.singularity.")
+        || name.startsWith("com.yourkit.")
+        || name.startsWith("com.cisco.")) {
```

Comment:
`com.cisco` is too restrictive. Imagine someone from cisco using this agent.  Look at the upstream class how it excluded new relic classes - the same story they wanted to use the agent internally so the agent had to exclude only NR agent classes.

Reply:
Good point! I'll update this to be `com.cisco.mtagent.` to match the `com.cisco.mtagent.entry.MTAgent$MultiTenantAgentClassLoader`